### PR TITLE
Replaces some otherwise tricky language within the codebase.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Replaces all instances of the word imposter with the term "lookalike", and the word among with the word "alongside".

## Why It's Good For The Game

Well, uhhh.

At least the second one was discussed in the community meeting? Helps distance us away from that other social deduction game I suppose.

Look I'm gonna be real with you I'm having second thoughts oh well smashing that create pull request button.

## Changelog not required as there are no player facing changes.

### CODE REVIEW

**Comment by YakumoChen** [comment]
Most of these substitutions are pretty grammatically bad...

**Comment by YakumoChen** [comment]
If you're gonna do this (and I really don't see the big deal with the word "among", go back and at least read what you're replacing, there's too many instances to pore over

File: code/modules/vehicles/mecha/combat/reticence.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/vehicle/sealed/mecha/combat/reticence
-	desc = "A silent, fast, and nigh-invisible miming exosuit. Popular among mimes and mime assassins."
+	desc = "A silent, fast, and nigh-invisible miming exosuit. Popular alongside mimes and mime assassins."
```
Comment (YakumoChen) [inline code/modules/vehicles/mecha/combat/reticence.dm]: "with"

File: code/modules/vehicles/mecha/combat/marauder.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /obj/vehicle/sealed/mecha/combat/marauder
-	desc = "Heavy-duty, combat exosuit, developed after the Durand model. Rarely found among civilian populations."
+	desc = "Heavy-duty, combat exosuit, developed after the Durand model. Rarely found alongside civilian populations."
 	name = "\improper Marauder"
```
Comment (YakumoChen) [inline code/modules/vehicles/mecha/combat/marauder.dm:3]: Within

File: code/modules/vehicles/mecha/combat/gygax.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /obj/vehicle/sealed/mecha/combat/gygax
-	desc = "A lightweight, security exosuit. Popular among private and corporate security."
+	desc = "A lightweight, security exosuit. Popular alongside private and corporate security."
```
Comment (YakumoChen) [inline code/modules/vehicles/mecha/combat/gygax.dm]: Among is about the best word to use here.

File: code/modules/research/machinery/_production.dm [hunk 4]
```diff
@@ -10,7 +10,7 @@
 	var/allowed_buildtypes = NONE
 	var/list/datum/design/cached_designs
 	var/list/datum/design/matching_designs
-	var/department_tag = "Unidentified" //used for material distribution among other things.
+	var/department_tag = "Unidentified" //used for material distribution alongside other things.
```
Comment (YakumoChen) [inline code/modules/research/machinery/_production.dm]: Amongst

File: code/modules/surgery/bodyparts/parts.dm [hunk 5]
```diff
@@ -154,7 +154,7 @@
 /obj/item/bodypart/r_arm
 	name = "right arm"
 	desc = "Over 87% of humans are right handed. That figure is much lower \
-		among humans missing their right arm."
+		alongside humans missing their right arm."
 	icon_state = "default_human_r_arm"
```
Comment (YakumoChen) [inline code/modules/surgery/bodyparts/parts.dm:158]: Nonsensical substitute
"In only"?

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm [hunk 6]
```diff
@@ -99,7 +99,7 @@
 	melee_damage_lower = 15
 	melee_damage_upper = 20
 	light_color = LIGHT_COLOR_PURPLE
-	deathmessage = "collapses into a pile of bones, their suit dissolving among the plasma!"
+	deathmessage = "collapses into a pile of bones, their suit dissolving alongside the plasma!"
```
Comment (YakumoChen) [inline code/modules/mob/living/simple_animal/hostile/skeleton.dm:102]: With

File: code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm [hunk 7]
```diff
@@ -29,7 +29,7 @@ Difficulty: Hard
 
 /mob/living/simple_animal/hostile/megafauna/bubblegum
 	name = "bubblegum"
-	desc = "In what passes for a hierarchy among slaughter demons, this one is king."
+	desc = "In what passes for a hierarchy alongside slaughter demons, this one is king."
```
Comment (YakumoChen) [inline code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm]: Among

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 8]
```diff
@@ -191,7 +191,7 @@
 	SSblackbox.record_feedback("nested tally", "item_used_for_combat", 1, list("[I.force]", "[I.type]"))
 	SSblackbox.record_feedback("tally", "zone_targeted", 1, target_area)
 
-	// the attacked_by code varies among species
+	// the attacked_by code varies alongside species
```
Comment (YakumoChen) [inline code/modules/mob/living/carbon/human/human_defense.dm]: Between

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 9]
```diff
@@ -375,7 +375,7 @@
 
 /obj/item/reagent_containers/food/drinks/bottle/lizardwine
 	name = "bottle of lizard wine"
-	desc = "An alcoholic beverage from Space China, made by infusing lizard tails in ethanol. Inexplicably popular among command staff."
+	desc = "An alcoholic beverage from Space China, made by infusing lizard tails in ethanol. Inexplicably popular alongside command staff."
```
Comment (YakumoChen) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: Within

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 10]
```diff
@@ -255,7 +255,7 @@
 /obj/item/paper/fluff/awaymissions/snowdin/profile/overseer
 	name = "Personnel Record AOP#01"
 	info = {"<b><center>Personnel Log</b></center><br><br><b>Name:</b>Caleb Reed<br><b>Age:</b>38<br><b>Gender:</b>Male<br><b>On-Site Profession:</b>Outpost Overseer<br><br><center><b>Information</b></center><br><center>Caleb Reed lead several expeditions
-		among uncharted planets in search of plasma for Nanotrasen, scouring from hot savanas to freezing arctics. Track record is fairly clean with only incidient including the loss of two researchers during the
+		alongside uncharted planets in search of plasma for Nanotrasen, scouring from hot savanas to freezing arctics. Track record is fairly clean with only incidient including the loss of two researchers during the
 		expedition of <b>_______</b>, where mis-used of explosive ordinance for tunneling causes a cave-in."}
```
Comment (YakumoChen) [inline code/modules/awaymissions/mission_code/snowdin.dm:259]: Of

File: code/modules/awaymissions/mission_code/caves.dm [hunk 11]
```diff
@@ -50,7 +50,7 @@
 
 /obj/item/paper/fluff/awaymissions/caves/safety_notice
 	name = "safety notice"
-	info = "<center>Some of the miners have gone to laying some mine traps among the lower levels of the mine to keep the monsters at bay.  This probably isn't the smartest idea in a cavern like this but the boys seem to get a chuckle out of every distant blast they hear go off, so I guess it works </center>"
+	info = "<center>Some of the miners have gone to laying some mine traps alongside the lower levels of the mine to keep the monsters at bay.  This probably isn't the smartest idea in a cavern like this but the boys seem to get a chuckle out of every distant blast they hear go off, so I guess it works </center>"
```
Comment (YakumoChen) [inline code/modules/awaymissions/mission_code/caves.dm]: Within

**Comment by ArcaneMusic** [comment]
> If you're gonna do this (and I really don't see the big deal with the word "among", go back and at least read what you're replacing, there's too many instances to pore over

No no, you're right, just took me a moment since this was mostly for the shock value

File: code/modules/antagonists/wizard/equipment/spellbook.dm [hunk 12]
```diff
@@ -118,7 +118,7 @@
 
 /datum/spellbook_entry/fireball
 	name = "Fireball"
-	desc = "Fires an explosive fireball at a target. Considered a classic among all wizards."
+	desc = "Fires an explosive fireball at a target. Considered a classic alongside all wizards."
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
```
Comment (YakumoChen) [inline code/modules/antagonists/wizard/equipment/spellbook.dm:122]: Amongst

File: code/game/objects/structures/bedsheet_bin.dm [hunk 13]
```diff
@@ -562,12 +562,12 @@ LINEN BINS
...
 
-	else if(amount && !hidden && I.w_class < WEIGHT_CLASS_BULKY) //make sure there's sheets to hide it among, make sure nothing else is hidden in there.
+	else if(amount && !hidden && I.w_class < WEIGHT_CLASS_BULKY) //make sure there's sheets to hide it alongside, make sure nothing else is hidden in there.
 		if(!user.transferItemToLoc(I, src))
-			to_chat(user, span_warning("\The [I] is stuck to your hand, you cannot hide it among the sheets!"))
+			to_chat(user, span_warning("\The [I] is stuck to your hand, you cannot hide it alongside the sheets!"))
 			return
 		hidden = I
-		to_chat(user, span_notice("You hide [I] among the sheets."))
+		to_chat(user, span_notice("You hide [I] alongside the sheets."))
```
Comment (YakumoChen) [inline code/game/objects/structures/bedsheet_bin.dm]: Between

File: code/game/objects/structures/bedsheet_bin.dm [hunk 14]
```diff
@@ -562,12 +562,12 @@ LINEN BINS
 			new /obj/item/stack/rods(loc, 2)
 			qdel(src)
 
-	else if(amount && !hidden && I.w_class < WEIGHT_CLASS_BULKY) //make sure there's sheets to hide it among, make sure nothing else is hidden in there.
+	else if(amount && !hidden && I.w_class < WEIGHT_CLASS_BULKY) //make sure there's sheets to hide it alongside, make sure nothing else is hidden in there.
 		if(!user.transferItemToLoc(I, src))
-			to_chat(user, span_warning("\The [I] is stuck to your hand, you cannot hide it among the sheets!"))
+			to_chat(user, span_warning("\The [I] is stuck to your hand, you cannot hide it alongside the sheets!"))
```
Comment (YakumoChen) [inline code/game/objects/structures/bedsheet_bin.dm]: Between

File: code/game/objects/structures/bedsheet_bin.dm [hunk 15]
```diff
@@ -133,7 +133,7 @@ LINEN BINS
 
 /obj/item/bedsheet/captain
 	name = "captain's bedsheet"
-	desc = "It has a Nanotrasen symbol on it, and was woven with a revolutionary new kind of thread guaranteed to have 0.01% permeability for most non-chemical substances, popular among most modern captains."
+	desc = "It has a Nanotrasen symbol on it, and was woven with a revolutionary new kind of thread guaranteed to have 0.01% permeability for most non-chemical substances, popular alongside most modern captains."
```
Comment (YakumoChen) [inline code/game/objects/structures/bedsheet_bin.dm]: With

File: code/game/objects/structures/barsigns.dm [hunk 16]
```diff
@@ -283,7 +283,7 @@
 /datum/barsign/the_lightbulb
 	name = "The Lightbulb"
 	icon = "the_lightbulb"
-	desc = "A cafe popular among moths and moffs. Once shut down for a week after the bartender used mothballs to protect her spare uniforms."
+	desc = "A cafe popular alongside moths and moffs. Once shut down for a week after the bartender used mothballs to protect her spare uniforms."
```
Comment (YakumoChen) [inline code/game/objects/structures/barsigns.dm:287]: With or among

File: code/game/objects/items/food/snacks.dm [hunk 17]
```diff
@@ -93,7 +93,7 @@
 
 /obj/item/food/chips/shrimp
 	name = "shrimp chips"
-	desc = "Deep-fried, shrimp flavored chips. A favorite junkfood among seafood connoisseurs!"
+	desc = "Deep-fried, shrimp flavored chips. A favorite junkfood alongside seafood connoisseurs!"
```
Comment (YakumoChen) [inline code/game/objects/items/food/snacks.dm]: Among

File: code/game/objects/items/spear.dm [hunk 18]
```diff
@@ -146,7 +146,7 @@
 //GREY TIDE
 /obj/item/spear/grey_tide
 	name = "\improper Grey Tide"
-	desc = "Recovered from the aftermath of a revolt aboard Defense Outpost Theta Aegis, in which a seemingly endless tide of Assistants caused heavy casualities among Nanotrasen military forces."
+	desc = "Recovered from the aftermath of a revolt aboard Defense Outpost Theta Aegis, in which a seemingly endless tide of Assistants caused heavy casualities alongside Nanotrasen military forces."
```
Comment (YakumoChen) [inline code/game/objects/items/spear.dm]: To

File: code/datums/ruins/space.dm [hunk 19]
```diff
@@ -272,7 +272,7 @@
 	id = "whiteshipruin_box"
 	suffix = "whiteshipruin_box.dmm"
 	name = "NT Medical Ship"
-	description = "An ancient ship, said to be among the first discovered derelicts near Space Station 13 that was still in working order. \
+	description = "An ancient ship, said to be alongside the first discovered derelicts near Space Station 13 that was still in working order. \
```
Comment (YakumoChen) [inline code/datums/ruins/space.dm]: Amongst

File: code/datums/quirks/neutral.dm [hunk 20]
```diff
@@ -326,7 +326,7 @@
 
 /datum/quirk/item_quirk/photographer
 	name = "Photographer"
-	desc = "You carry your camera and personal photo album everywhere you go, and your scrapbooks are legendary among your coworkers."
+	desc = "You carry your camera and personal photo album everywhere you go, and your scrapbooks are legendary alongside your coworkers."
```
Comment (YakumoChen) [inline code/datums/quirks/neutral.dm]: Among

**Review by YakumoChen — CHANGES_REQUESTED** [review]

**Comment by Mothblocks** [comment]
image

**Comment by ArcaneMusic** [comment]
> image

I limited it to .dm files to avoid total societal collapse

Also I'm truly sorry yakumo I was going to leave it until after I ate dinner but damn

**Comment by YakumoChen** [comment]
Some of yours are better anyway 
This is too much effort for a shitty meme pr but I digress

**Comment by Ghommie** [comment]
Wow, someone is making something out of nothing here. And I don't even get how this is good for the game from the main post.

**Comment by Ryll-Ryll** [comment]
funny, but this sets a bad precedent for future meme-of-the-week grep replacements

**Comment by ArcaneMusic** [comment]
Based, thanks chief

**Pull request closed without merging**